Argo/Types/Decoded.swift

```diff
@@ -85,3 +85,10 @@ public extension Decoded {
     }
   }
 }
+
+public func ?? <T>(lhs: Decoded<T>, rhs: T) -> T {
```

Comment:
ya I agree ... I don't like those file names but I didn't have a better idea at the time ... I'm open to suggestions